#ifndef TAB1_H
#define TAB1_H

#include <QWidget>
#include<QToolButton>
#include<QToolBar>
namespace Ui {
class Tab1;
}

class Tab1 : public QWidget
{
    Q_OBJECT

public:
    explicit Tab1(QWidget *parent = nullptr);
    ~Tab1();
    QToolButton *button1;
    QToolButton *button2;
    QToolButton *button3;
    QToolButton *button4;
    QToolButton *button5;
    QToolButton *button6;
    QToolButton *button7;
    QToolButton *button8;
    QToolButton *button9;
    QToolButton *button10;
    QToolButton *button11;
    QToolButton *button12;
    QToolButton *button13;
    QToolButton *button14;
    QToolButton *button15;
    QToolButton *button16;
    QToolButton *button17;
    QToolButton *button18;
    QToolButton *button19;
    QToolButton *button20;
    QToolButton *button21;
    QToolButton *button22;
    QToolButton *button23;
    QToolButton *button24;
    QToolButton *button25;
    QToolButton *button26;
    QToolButton *button27;
    QToolButton *button28;

private:
    Ui::Tab1 *ui;
};

#endif // TAB1_H
